elasticdl/python/data/reader/csv_reader.py

```diff
@@ -0,0 +1,70 @@
+import csv
+
+import numpy as np
+import tensorflow as tf
+
+from elasticdl.python.data.reader.data_reader import (
+    AbstractDataReader,
+    Metadata,
+)
+
+
+class CSVDataReader(AbstractDataReader):
+    """This reader is used to read data from a csv file. It is convenient for
+    user to locally run and debug a Keras model by using this reader.
+    However, it can not be used with distribution strategy because it can not
+    read data by line indices.
+    """
+
+    def __init__(self, **kwargs):
+        """
+        Args:
+            kwargs should contains "seq" and "columns" like
+            'seq=",",column=["sepal.length", "sepal.width", "variety"]'
+        """
+        AbstractDataReader.__init__(self, **kwargs)
+        self.seq = kwargs.get("seq", ",")
+        self.selected_columns = kwargs.get("columns", None)
+
+    def read_records(self, task):
+        with open(task.shard_name, "r", encoding="utf-8") as csv_file:
+            csv_reader = csv.reader(csv_file, delimiter=self.seq)
+            csv_columns = next(csv_reader)
+            selected_columns = (
+                csv_columns
+                if self.selected_columns is None
+                else self.selected_columns
+            )
+            if not set(csv_columns) >= set(selected_columns):
```

Comment:
`if not set(csv_columns) >= set(selected_columns):` to

`if set(selected_columns) > set(csv_columns)`

Reply:
I think for set comparision:

if not set(csv_columns) >= set(selected_columns): is not equal to 
if set(selected_columns) > set(csv_columns)

set(csv_columns) >= set(selected_columns) 
set(csv_columns) < set(selected_columns) 
can be false at the same time

Reply:
Hmm, interesting. 
Maybe `if not set(selected_columns). issubset(set(csv_columns))` is more readable?

Reply:
> csv_columns

I think so

Reply:
> Hmm, interesting.
> Maybe `if not set(selected_columns). issubset(set(csv_columns))` is more readable?

I think so

Reply:
Yes,` if not set(csv_columns) >= set(selected_columns)` is not equal to
`if set(selected_columns) > set(csv_columns)`. For example, `not set([1, 3, 4]) >= set([1, 5])` return True, but `set([1, 5]) > set([1, 3, 4])` return False. The operator "set_a >= set_b" represents that set_b is a subset of set_a.

Reply:
> Hmm, interesting.
> Maybe `if not set(selected_columns). issubset(set(csv_columns))` is more readable?

Yes!